lib/variable/test/variable_keyword_args_constructor_test.cpp

```diff
@@ -67,10 +67,6 @@ TEST(CreateVariableTest, from_vector) {
   EXPECT_EQ(makeVariable<double>(Dims{Dim::X}, Shape{3},
                                  Values(std::vector<int>{1, 2, 3})),
             makeVariable<double>(Dims{Dim::X}, Shape{3}, Values{1, 2, 3}));
-
-  const std::vector<double> v{1, 2, 3};
-  auto varRef = makeVariable<double>(Dims{Dim::X}, Shape{3}, Values{1, 2, 3});
-  auto var = makeVariable<double>(Dims{Dim::X}, Shape{3}, Values(v));
```

Comment:
This might have served as a test that it compiles? Maybe it should not be removed?

Reply:
Or it is a leftover from a refactor? I can put it back and add an equivalent assertion to the one above.